source/opt/wrap_opkill.cpp

```diff
@@ -105,7 +110,7 @@ uint32_t WrapOpKill::GetVoidFunctionTypeId() {
   return type_mgr->GetTypeInstruction(&func_type);
 }
 
-uint32_t WrapOpKill::GetOpKillFuncId() {
+uint32_t WrapOpKill::GetOpKillFuncId(Instruction* dbg_info) {
```

Comment:
I don't think we should do this change.  It is a tough one because we want a single OpKill Function, which can be called in many places.  In this case, you will use the debug info for the first location of an OpKill, but that could be completely wrong for the next instance.

I would prefer to leave the OpKill with no debug info.  The debug info on the call is more informative.

Reply:
Ok. I add the debug information only for `OpFunctionCall` for the new function that executes `OpKill`.